ansible/group_vars/all/ceos.yml

```diff
@@ -1,4 +1,4 @@
-ceos_image_filename: cEOS64-lab-4.23.2F.tar.xz
-ceos_image_orig: ceosimage:4.23.2F
-ceos_image: ceosimage:4.23.2F-1
```

Comment:
instead of remove them, maybe we can comment out them, so that users can still uncomment them and use them.

Reply:
Greate advice, modified